.storybook/preview.js

```diff
@@ -44,4 +46,32 @@ export const parameters = {
   },
 };
 
+WebComponentFormatter(customElements);
+
 setCustomElements(customElements);
+
+function WebComponentFormatter(customElements) {
```

Comment:
@madsrasmussen especially take note of this function if you see anything that could break! It is very hacky ;-)